src/views/Header.js

```diff
@@ -99,11 +80,63 @@ class Header extends React.Component<DefaultProps, HeaderProps, *> {
     );
   }
 
+  _getHeaderTitle(navigation: Navigation): ?string {
+    const header = this.props.router.getScreenConfig(navigation, 'header');
+    let title = null;
+    if (header && header.title) {
+      title = header.title;
+    } else {
+      title = this.props.router.getScreenConfig(navigation, 'title');
+    }
+    if (typeof title === 'string') {
```

Comment:
This also needs a check. Technically speaking this will never be reached since `CardStack` provides methods that check for `React.Element`